Gemfile

```diff
@@ -13,11 +13,18 @@ gem 'sqlite3'
 # Use Puma as the app server
 gem 'puma', '~> 3.7'
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
+gem 'sass-rails', '>= 3.2'
```

Comment:
So, the change from 5.0 down to 3.2 was an effort to debug and get sass/bootstrap working. In case we run into errors with 3.2 or want to use 5.0, this will be the place to change it.